front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -360,6 +368,16 @@ export default function ActionsScreen({
                 Before replying, the agent can use multiple of those tools to
                 gather information and provide you with the best possible
                 answer.
+                <br />
```

Comment:
It feels weird to have two dropdown options on the left + one drop option on the right and `Read our guide`  button in between. I checked with Ed and moved it in the paragraph like we do in Spaces.